lib/index.js

```diff
@@ -58,9 +58,24 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         }]
       };
       // actor is not required, but passed through if given
-      const {actor = null} = req.user || {};
+      let {actor = null} = req.user || {};
       await brAccount.insert({actor, account, meta});
-      res.status(201).location(baseUri + '/' + account.id).json(account);
+
+      if(actor || !cfg.autoLoginNewAccounts) {
+        // actor given or auto-login disabled, do not auto-login the new account
+        res.status(201).location(baseUri + '/' + account.id).json(account);
+        return;
+      }
+
+      // no actor given, so auto-login
+      actor = await brAccount.getCapabilities({id: account.id});
+      const user = {account, actor};
+      req.logIn(user, err => {
```

Comment:
Might look nicer with `await` and `promisify` to get rid of the `next(err)` part since this is in an `asyncHandler`.